app.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+}
+
+var newText = document.createTextNode('text');
+var tableRow = document.createElement('tr');
+var tableHeader = document.createElement('th');
+var tableData = document.createElement ('td');
```

Comment:
Move these four var definitions into the functions that use them.  This will remove the vars from the global namespace and only use the memory needed when the function they are used in is run.